src/passwordgenerator/PasswordGenerator.java

```diff
@@ -0,0 +1,14 @@
+package src.passwordgenerator;
+
+import src.UI.base_UI;
+
+public class PasswordGenerator {
+
+    PasswordGenerator() {
+        base_UI system_dialogue = new base_UI();
```

Comment:
Use a consistent case for naming variables. Usually variables in Java follow a `Camel Case`.
Refer: https://www.theserverside.com/feature/A-guide-to-common-variable-naming-conventions